node_helper.js

```diff
@@ -15,15 +15,34 @@ module.exports = NodeHelper.create({
 		if(notification === "GET_COMIC"){
 			
 			var comicJsonUri = payload.config.dailyJsonUrl;
+			var comic;
+			var rndcomic;
+			var rndUrl;
+			var body;
+			var d = new Date();
+			var n = d.getDay(); 
```

Comment:
In association with the comment below, I'd recommend moving these local vars into the callback function within request(). They're only ever used there and it would get rid of the ambiguity of "this".